data_steward/tools/clean_fitbit.sh

```diff
@@ -78,24 +77,24 @@ export BIGQUERY_DATASET_ID="${fitbit_dataset}"
 export PYTHONPATH="${PYTHONPATH}:${CLEAN_DEID_DIR}:${DATA_STEWARD_DIR}"
 
 # create empty fitbit de-id dataset
-bq mk --dataset --description "${dataset_release_tag} de-identified version of ${fitbit_dataset}" "${APP_ID}":"${registered_fitbit_deid}"
-"${TOOLS_DIR}"/table_copy.sh --source_app_id "${APP_ID}" --target_app_id "${APP_ID}" --source_dataset "${fitbit_dataset}" --target_dataset "${registered_fitbit_deid}"
+bq mk --dataset --description "${dataset_release_tag} de-identified version of ${fitbit_dataset}" --label "phase:clean" --label "release_tag:${dataset_release_tag}" --label "de_identified:true"  "${APP_ID}":"${fitbit_deid_dataset}"
```

Comment:
I would actually think the phase would be `staging` at this point and de_identified would be `false`, because we haven't actually cleaned the data yet.